src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +100,36 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files for classifications. These classifications within each file get added to the overview-features page as metadata.
+     * File and metadata order within the individual files are preserved when classifications are added.
+     *
+     * @param propertiesFiles
+     *            property files to read
+     */
+    public void parseClassificationsFiles(List<String> propertiesFiles) {
```

Comment:
I like it - method name is more abut business needs and the parameter is more about technical expectation